wormhole-connect/src/components/FooterNavBar.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { makeStyles } from 'tss-react/mui';
+import { ICON } from 'utils/style';
+import { Route, setRoute } from 'store/router';
+
+const useStyles = makeStyles()(() => ({
+  menuIcon: ICON,
+  menu: {
+    display: 'flex',
+    flexDirection: 'row',
+    gap: '8px',
+    padding: '8px',
```

Comment:
May I suggest a few tweaks for this css to space the items around a bit more uniformly?

```suggestion
    padding: '8px',
    width: '85%',
```

Reply:
applied